lake_hylia.logic

```diff
@@ -1,3 +1,93 @@
 area LakeHylia:
-  room BeanstalkArea
-  room Main
+  room BeanstalkArea:
+    node EastArea:
+      door LeftLadder
+      door RightLadder
+    node WestArea:
+      door Ladder
+    node BeanstalkFusion:
+      entrance beanStalkFusion
+    node BeanStalkFusionExit:
+      exit beanStalkFusion
+    BeanStalkFusionExit -> WestArea: flag HyliaBeanstalkFusion
+    WestArea -> BeanstalkFusion: flag HyliaBeanstalkFusion
+    node EastArea +: exit Virtual.Zeffa.FluteAway
+    node WestArea +: exit Virtual.Zeffa.FluteAway
+  room Main:
+    node Crest
+      entrance Crest
```

Comment:
This is wrong in the minish woods example too, but this is something where you enter from somewhere else - i.e. an exit.
```suggestion
      exit Crest
```